src/main/kotlin/org/jetbrains/research/transformations/Main.kt

```diff
@@ -1,5 +1,48 @@
 package org.jetbrains.research.transformations
 
+import com.github.gumtreediff.gen.python.PythonTreeGenerator
+import com.github.gumtreediff.tree.ITree
+import com.github.gumtreediff.tree.TreeContext
+import java.io.File
+import org.jetbrains.research.transformations.Transformations.*
+
+fun setupParser() {
+    //copy parser file into /tmp directory, required by Gumtree
+    val sourcePath :String = "src/main/resources/pythonparser"
+    val targetPath :String = "/tmp/pythonparser"
+    val pythonparserFile :File = File(sourcePath)
+
+    try {
+        val targetFile :File = File(targetPath)
+        pythonparserFile.copyTo(targetFile)
+        val command = arrayOf("chmod", "+x", targetFile.getAbsolutePath())
+        val b = ProcessBuilder(*command)
+        b.directory(targetFile.parentFile)
+        b.start()
+    } catch (e: kotlin.io.FileAlreadyExistsException) {
+        println("Pythonparser file already exists in ${targetPath} directory")
```

Comment:
It would be better to use **logging** for such things instead of printing to the console, but since we didn't tell you about it, we can explain it in the upcoming meeting, or you can google it a little :)

Reply:
I agree with it! It is a great idea!